src/server/index.jsx

```diff
@@ -73,6 +73,12 @@ if (process.env.APP_ENV === 'local') {
     });
   };
 
+  server.use((req, res, next) => {
```

Comment:
Tiny thing, but could you put this in the chain of them on line 82, they should all chain off each-other

`service.use().get().get()` etc